wodles/azure/tests/test_azure.py

```diff
@@ -627,30 +627,31 @@ def test_start_storage_ko_credentials(mock_logging):
     mock_logging.assert_called_once()
 
 
-@pytest.mark.parametrize('blob_date, min_date, max_date, desired_date, extension, reparse, json_file, inline, send_events', [
+@pytest.mark.parametrize(
+    'blob_date, min_date, max_date, desired_date, extension, reparse, json_file, inline, send_events, prefix', [
     # blob_date < desired_date - Blobs should be skipped
-    (PRESENT_DATE, PAST_DATE, PAST_DATE, FUTURE_DATE, None, False, False, False, False),
+    (PRESENT_DATE, PAST_DATE, PAST_DATE, FUTURE_DATE, None, False, False, False, False, None),
     # blob_date > desired_date, min_date == blob_date and blob_date < max_date - Blobs should be skipped
-    (PRESENT_DATE, PRESENT_DATE, FUTURE_DATE, PAST_DATE, None, False, False, False, False),
+    (PRESENT_DATE, PRESENT_DATE, FUTURE_DATE, PAST_DATE, None, False, False, False, False, None),
     # blob_date > desired_date, min_date < blob_date and blob_date == max_date - Blobs should be skipped
-    (FUTURE_DATE, PRESENT_DATE, FUTURE_DATE, PAST_DATE, None, False, False, False, False),
+    (FUTURE_DATE, PRESENT_DATE, FUTURE_DATE, PAST_DATE, None, False, False, False, False, None),
     # blob_date > desired_date, min_date < blob_date and blob_date < max_date - Blobs should be skipped
-    (PAST_DATE, PRESENT_DATE, FUTURE_DATE, PAST_DATE, None, False, False, False, False),
+    (PAST_DATE, PRESENT_DATE, FUTURE_DATE, PAST_DATE, None, False, False, False, False, None),
     # blob_date < min_datetime - Blobs must be processed
-    (PAST_DATE, PRESENT_DATE, FUTURE_DATE, PAST_DATE, None, False, False, False, True),
+    (PAST_DATE, PRESENT_DATE, FUTURE_DATE, PAST_DATE, None, False, False, False, True, None),
     # blob_date > max_datetime - Blobs must be processed
-    (FUTURE_DATE, PAST_DATE, PRESENT_DATE, FUTURE_DATE, None, False, False, True, True),
+    (FUTURE_DATE, PAST_DATE, PRESENT_DATE, FUTURE_DATE, None, False, False, True, True, None),
     # Reparse old logs
-    (FUTURE_DATE, FUTURE_DATE, FUTURE_DATE, FUTURE_DATE, None, True, False, True, True),
+    (FUTURE_DATE, FUTURE_DATE, FUTURE_DATE, FUTURE_DATE, None, True, False, True, True, None),
     # Only .json files must be processed
-    (FUTURE_DATE, PAST_DATE, PRESENT_DATE, FUTURE_DATE, ".json", False, False, False, True),
-    (FUTURE_DATE, PAST_DATE, PRESENT_DATE, FUTURE_DATE, ".json", False, False, True, True),
-    (FUTURE_DATE, PAST_DATE, PRESENT_DATE, FUTURE_DATE, ".json", False, True, False, True),
+    (FUTURE_DATE, PAST_DATE, PRESENT_DATE, FUTURE_DATE, ".json", False, False, False, True, None),
+    (FUTURE_DATE, PAST_DATE, PRESENT_DATE, FUTURE_DATE, ".json", False, False, True, True, None),
+    (FUTURE_DATE, PAST_DATE, PRESENT_DATE, FUTURE_DATE, ".json", False, True, False, True, None),
```

Comment:
There is no need to add an additional parameter if the value is going to be always `None`. Remove them and update the `assert_called_with` accordingly.